.pylintrc

```diff
@@ -224,7 +224,8 @@ logging-modules=logging
 [MISCELLANEOUS]
 
 # List of note tags to take in consideration, separated by a comma.
-notes=FIXME,XXX,TODO
+# notes=FIXME,XXX,TODO
+notes=FIXME,XXX
```

Comment:
I see you changed the pylint configuration to _not_ issue a warning for `TODO` tags. I don't think this is a good choice - future tasks should appear in issues, not the code.

Reply:
Reverted this change :)

Reply:
Thanks!